modules/combined/tests/creep_plasticity_combined/combined_creep_plasticity.i

```diff
@@ -9,7 +9,6 @@
 #   further yielding during unloading. The initial yield stress (sigma_0) is prescribed
 #   as 20 with a plastic strain hardening of 100. The mesh is a 1x1x1 cube with symmetry
 #   boundary conditions on three planes to provide a uniaxial stress field.
-#   The temperature is held constant at 1000.
```

Comment:
Sure--I'd be glad to move these tests to tensor mechanics.  The analytical solution from the paper does not include temperature, so the temperature was held constant to avoid making an impact.  Should I move the solid mechanics tests to solid mechanics or just keep them here temporarily?

The isotropic plasticity model only uses the temperature if a yield stress function is provided (here one is not), and the creep model uses the temperature in the activation term, which we've essentially cancelled out in these input file by setting the activation energy Q to 0 to match the journal paper.